browser/ui/webui/brave_rewards_internals_ui.cc

```diff
@@ -12,17 +12,16 @@
 #include "brave/components/brave_rewards/browser/rewards_service.h"
 #include "brave/components/brave_rewards/browser/rewards_service_factory.h"
 #include "brave/components/brave_rewards/common/pref_names.h"
+#include "brave/components/brave_rewards/resources/grit/brave_rewards_internals_generated_map.h"
```

Comment:
was a dep added for this? Make sure to add deps for any headers you add

Reply:
yes, this include works now for bot desktop and Android and moved up from line 22.